eng/pipelines/prepare-pipelines.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+
+extends:
+  template: /eng/common/pipelines/templates/steps/prepare-pipelines.yml
+  parameters:
+    Repository: $(Build.Repository.Name)
+    Prefix: java
+    CIConventionOptions: ''
+    UPConventionOptions: '--variablegroup 1 --variablegroup 58 --variablegroup 76 --variablegroup 56'
```

Comment:
I saw you are setting variable group here, but do they show up in pipeline?

Reply:
These are the variable groups that we already use in the overall pipeline generation job that runs every day, they are copied from there. The variable groups contain the secrets that the release pipelines need for publishing etc.